packages/common/react-components/src/config/tailwind.ts

```diff
@@ -246,5 +246,8 @@ export const tailwindConfig = ({
   },
   plugins: [tailwindcssLogical, tailwindcssForms, tailwindcssRadix()],
   ...(env === 'development' && { mode: 'jit' }),
```

Comment:
what is jit?

Reply:
@richburdon https://v2.tailwindcss.com/docs/just-in-time-mode